### PR TITLE
add travis.yml

### PR DESCRIPTION
**Pull request by kumagi**

### CODE REVIEW

File: .travis/install_openssl.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+OPENSSL="OpenSSL_1_1_1c"
+OPENSSL_ZIP="${OPENSSL}.zip"
+travis_retry wget --no-check-certificate https://github.com/openssl/openssl/archive/${OPENSSL_ZIP}
+echo "e519cd282e7a94ea1d72f223ba792ad0  ${OPENSSL_ZIP}" > openssl_md5.txt
+md5sum -c openssl_md5.txt
+unzip ${OPENSSL_ZIP}
+pushd openssl-${OPENSSL}
```
Comment (tomokinat) [inline .travis/install_openssl.sh]: completely optional: indent like
```sh
pushd
  # do something
  # another
  # yet another
popd
```

File: .travis/install_openssl.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+OPENSSL="OpenSSL_1_1_1c"
+OPENSSL_ZIP="${OPENSSL}.zip"
+travis_retry wget --no-check-certificate https://github.com/openssl/openssl/archive/${OPENSSL_ZIP}
```
Comment (tomokinat) [inline .travis/install_openssl.sh]: Can't this be solved in another way? Just disabling cert check doesn't sound a good idea.

File: .travis/install_openssl.sh (hunk 1 above)
Comment (kumagi) [reply .travis/install_openssl.sh]: OK.

File: .travis/install_openssl.sh (hunk 2 above)
Comment (kumagi) [reply .travis/install_openssl.sh]: OK.

**Pull request merged**